src/bindings/rust/README.md

```diff
@@ -27,9 +27,63 @@ fn main() {
 }
 ```
 
+### Key
+
+A full example for using some of the key functionality. For all methods, see the documentation.
+
+```rust
+extern crate elektra;
+use elektra::{KeyBuilder, ReadableKey, StringKey, WriteableKey};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Can you put this example in a separate file (compiled with the build system)? Here in the tutorial there should be rather very small example codes with explanations.

Reply:
I shortened the example in the Readme to a very simple one and linked to the full example that is run together with `cargo test`, so is part of the build system.